packages/utils/log/README.md

```diff
@@ -0,0 +1,122 @@
+# Fusion Log Utilities
+
+This package provides a set of utilities for logging messages in a TypeScript/JavaScript application within the Fusion Framework. The primary purpose of this code is to provide a consistent and configurable logging mechanism that can be used throughout the application.
+
+## Overview
+
+The logging utilities in this package are designed to provide a flexible and extensible logging solution. The main components are:
+
+- `Logger` (abstract class): Defines the base interface and functionality for logging messages with different severity levels (debug, info, warn, error).
+- `ConsoleLogger`: A concrete implementation of the `Logger` class that logs messages to the console.
+- `LogLevel` (enum): Defines the different log levels that can be used to control the verbosity of logging.
+- `resolveLogLevel` (function): Resolves a log level from a string or number representation.
+
+## ConsoleLogger
+The `ConsoleLogger` class is a concrete implementation of the `Logger` class. It logs messages to the console using the appropriate console methods (`console.debug`, `console.info`, `console.warn`, `console.error`). The `ConsoleLogger` class also supports creating sub-loggers with a new title and optional subtitle.
+
+```typescript
+import { ConsoleLogger, LogLevel } from '@equinor/fusion-log';
+
+// Create a new ConsoleLogger instance
+const logger = new ConsoleLogger('MyApplication');
+
+// Set the logging level
+logger.level = LogLevel.Debug;
+
+// Log messages at different severity levels
+logger.debug('This is a debug message');
+logger.info('This is an info message');
+logger.warn('This is a warning message');
+logger.error('This is an error message');
+
+// Create a sub-logger
+const subLogger = logger.createSubLogger('SubComponent');
+subLogger.debug('This is a debug message from the sub-logger');
+```
+
+## LogLevel
+
+The `LogLevel` enum represents the different log levels that can be used to control the verbosity of logging. It includes the following levels:
+
+0. **None** - No logging will be performed.
+1. **Error** - Due to a more serious problem, the software has not been able to perform some function.
+2. **Warning** - An indication that something unexpected happened, or indicative of some problem in the near future.
+3. **Info** - Confirmation that things are working as expected.
+4. **Debug** - Detailed information, typically of interest only when diagnosing problems.
+
+#### Setting log level as Environment Variable
+
+The logging level can be configured using the `FUSION_LOG_LEVEL` environment variable. If the environment variable is set, the resolveDefaultLogLevel function will attempt to parse the log level from it. If the parsing fails, it will default to LogLevel.Debug in development environments or LogLevel.Error in production environments.
+
+> [!WARNING]
+> be sure to set `FUSION_LOG_LEVEL` in your package managers environment variables, before compiling code for production.'
```

Comment:
Because this is important, consider moving this warning higher up in the read-me and link to the `### Setting log level as Env Var` heading.